ui/src/app/pages/artifactVersion/components/modals/editMetaDataModal.tsx

```diff
@@ -192,13 +191,41 @@ export class EditMetaDataModal extends PureComponent<EditMetaDataModalProps, Edi
                 ...this.state.metaData,
                 labels
             }
+        }, () => {
+            this.validate();
         });
     };
 
     private onDescriptionChange = (value: string): void => {
         this.setSingleState("metaData", {
             ...this.state.metaData,
             description: value
+        }, () => {
+            this.validate();
+        });
+    };
+
+    private onPropertiesChange = (properties: ArtifactProperty[]): void => {
+        this.setSingleState("properties", properties, () => {
+            this.validate();
+        });
+    };
+
+    private validate = (): void => {
```

Comment:
I'm not sure if it's in the context of this change but I found it strange that I can add empty keys/empty values(and even both)
![Screenshot 2022-08-16 at 15 24 35](https://user-images.githubusercontent.com/5792097/184904503-7c3cd71d-7311-48dd-bf6e-0b889daa3b93.png)

Reply:
The one you mentioned plus "Empty key + empty value is allowed" is not ignored, it appears in the list.

Tested on Chrome

Reply:
OK I'll fix and push a new commit.

Reply:
I'm going to merge this PR and we can open a GH issue for the "Empty Key + Non-Empty Value" scenario if you can demo it for me tomorrow.

Reply:
No problem at all 👍